todo_app/app.py

```diff
@@ -1,11 +1,24 @@
-from flask import Flask
+import flask
```

Comment:
You could export multiple classes/functions here instead of the whole module if you find it more readable, but it's up to you.